terraform/modules/data-share-service/cloudwatch.tf

```diff
@@ -23,6 +23,30 @@ resource "aws_cloudwatch_dashboard" "metrics_dashboard" {
             [
               "gdxApp",
               "API_CALLS.IngestedEvents.count"
+            ],
+            [
+              ".",
+              "API_CALLS.CallsToLev.count"
+            ],
+            [
+              ".",
+              "API_CALLS.ResponsesFromLev.count"
+            ],
+            [
+              ".",
+              "API_CALLS.CallsToHmrc.count"
+            ],
+            [
+              ".",
+              "API_CALLS.ResponsesFromHmrc.count"
+            ],
+            [
+              ".",
+              "API_CALLS.CallsToPoll.count"
+            ],
+            [
+              ".",
+              "API_CALLS.CallsToEnrich.count"
             ]
           ],
           "period" : 300,
```

Comment:
We should also add this here as currently it is doing averages
    "stat": "Sum"